lib/src/main/kotlin/com/outdatedversion/survival/command/ReplyCommand.kt

```diff
@@ -0,0 +1,68 @@
+package com.outdatedversion.survival.command
+
+import co.aikar.commands.BaseCommand
+import co.aikar.commands.annotation.*
+import com.outdatedversion.survival.MessagingModule
+import com.outdatedversion.survival.Plugin
+import com.outdatedversion.survival.asText
+import com.outdatedversion.survival.model.PointOfInterest
+import com.outdatedversion.survival.model.asComponent
+import net.kyori.adventure.text.Component
+import net.kyori.adventure.text.format.NamedTextColor
+import net.kyori.adventure.text.format.TextDecoration
+import org.bukkit.Bukkit
+import org.bukkit.NamespacedKey
+import org.bukkit.entity.Player
+import org.bukkit.persistence.PersistentDataType
+import java.util.*
+
+@CommandAlias("r|reply|rpl")
+class ReplyCommand(private val module : MessagingModule
+
+): BaseCommand() {
+    @Default
+    @CommandCompletion("@nothing")
+    @Description("Replies a private message to the last player you messaged.")
+    @Syntax("<message>")
+    fun handleCommand(player: Player, vararg message: String) {
+      if(module.getReply(player.uniqueId) != null){
+          val recipientPlayer: Player? = Bukkit.getServer().getPlayer(module.getReply(player.uniqueId)!!)
+          if (recipientPlayer == null) {
+              player.sendMessage(
+                  Component.text(
+                      "The player is no longer online.",
```

Comment:
what do you think of something like "Nokoa is no longer online" instead?

Reply:
I was thinking about doing this, but did not because that would either require me to keep track of their names, or fetch the offline player with the saved UUIDs and get the name that way.

I will add this now, it will be using offline player using UUID to get the player name.